ElectronClient/gui/Header.jsx

```diff
@@ -100,7 +100,7 @@ class HeaderComponent extends React.Component {
 	}
 
 	determineButtonLabelState() {
-		const mediaQuery = window.matchMedia(`(max-width: ${550 * this.props.zoomFactor}px)`);
+		const mediaQuery = window.matchMedia(`(max-width: ${780 * this.props.zoomFactor}px)`);
 		const showButtonLabels = !mediaQuery.matches;
```

Comment:
Why did you change the value from 550 to 780?

Reply:
I've increased it to prevent the header content from overflowing. Like in the screenshot below:
![image](https://user-images.githubusercontent.com/26695184/76295124-0dac2700-62da-11ea-8f8e-72429195aed3.png)

Reply:
Hmm, in that case they should not overflow but the labels should be hidden that only the icons are shown.

Reply:
Yeah, if the media query doesn't match button labels are hidden.

Reply:
thx.